database/migrations/2021_02_23_185026_change_avatar_to_be_null_on_users.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php1
```

Comment:
```suggestion
<?php
```

![image](https://user-images.githubusercontent.com/4103693/116404738-3ae5cc80-a82f-11eb-8971-319f6f27982e.png)